src/accelerate/utils/modeling.py

```diff
@@ -386,7 +386,7 @@ def set_module_tensor_to_device(
         elif value is not None or not check_device_same(torch.device(device), module._parameters[tensor_name].device):
             param_cls = type(module._parameters[tensor_name])
             kwargs = module._parameters[tensor_name].__dict__
-            if param_cls.__name__ in ["Int8Params", "FP4Params"]:
+            if param_cls.__name__ in ["Int8Params", "FP4Params", "Params4bit"]:
```

Comment:
Given that we use `["Int8Params", "FP4Params", "Params4bit"]` already in [line 363](https://github.com/huggingface/accelerate/pull/2381/files#diff-d9b8e1f4731ccbfb7b66e8685b7dd7c500c1ae5bba510b610714e72a39049de0R363), how about putting this into a constant?